aws-api/src/test/java/com/amplifyframework/api/aws/AWSApiPluginTest.java

```diff
@@ -246,6 +255,13 @@ public void graphQlMutationGetsResponse() throws JSONException, ApiException {
 
         // Assert that the expected response was received
         assertEquals(expectedName, actualResponse.getData().getName());
+
+        // Verify that the expected hub event fired.
+        List<HubEvent<?>> events = networkStatusObserver.await();
```

Comment:
I used to have a convenient method on the `HubAccumulator` that allowed you to just get the first one. like `takeOne()`, it was called. Should we include an `awaitFirst()` to simplify the `List` return type? Maybe not. Just sayin'.

Reply:
That's a good idea. Just added a commit with that change.